library/models/library_editor_supplier.py

```diff
@@ -0,0 +1,24 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class LibraryEditorSupplier(models.Model):
+    """Defining Library Editor Supplier."""
+
+    _name = "library.editor.supplier"
```

Comment:
@ForamSerpentCS We can use res.partner directly here.
Please check.
cc @Trivedi-Vacha-SerpentCS

Reply:
We might not as res.partner object already has been used for Library Editor Supplier.